tests/model/test_model.py

```diff
@@ -1978,25 +1978,72 @@ def test__update_topic_index(
             "vary_each_msg",
             "visual_notification_status",
             "types_when_notify_called",
+            "is_muted_stream",
+            "is_muted_topic",
```

Comment:
Nit: These are setup values, so it would be clearer to have them before the `types_when_notify_called`.